axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/batch/BatchBillOfExchange.java

```diff
@@ -125,6 +120,17 @@ protected void createLCRAccountingMovesForInvoices(
     }
   }
 
+  @Transactional
```

Comment:
Rollback should be managed here

https://github.com/google/guice/wiki/Transactions#responding-to-exceptions